cmd/internal/cli/sign.go

```diff
@@ -27,7 +27,7 @@ var signSifGroupIDFlag = cmdline.Flag{
 	DefaultValue: uint32(0),
 	Name:         "group-id",
 	ShortHand:    "g",
-	Usage:        "group ID to be signed",
+	Usage:        "sign a specified group ID (default non)",
```

Comment:
I'm not sure this change really makes it any clearer _what_ the option does. Maybe something like:

"sign over multiple partitions/descriptors in the specified group"

Reply:
I changed it to:

```
sign all partitions in the specified group (default non)
```

Hows that?